### DISCUSSION TITLE
migrating to WordPress

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#4407] [Q&A] [answered]
my client wants to migrate his website to WordPress (due to having a visual editor for non coders) so can I migrate to WordPress by exporting or do I need a tool?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no automation tool for this as it will depend on your WordPress configuration and desired structure (especially if you have custom content types or use some 3rd party plugin).

You'll need to create manually SQL migration script that will transform your `pb_data/data.db` SQLite tables and records (_each PocketBase collection has a corresponding table_) into the specific WordPress db structure.
